mobile/native/src/health.rs

```diff
@@ -0,0 +1,65 @@
+use tokio::runtime::Runtime;
+use tokio::select;
+use tokio::sync::watch;
+
+use crate::event::EventInternal;
+use crate::event::{self};
+
+/// Services which status is monitored
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum Service {
+    Orderbook,
+}
+
+/// Health status of the node
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq)]
+pub enum ServiceStatus {
+    #[default]
+    Unknown,
+    Online,
+    Offline,
+}
+
+pub type ServiceUpdate = (Service, ServiceStatus);
```

Comment:
It actually had to become a struct, because although Dart can destruct a tuple without a problem, our event dispatch magic does not like this and I had to rework it :) 

will pull that part into this PR.